app/presenters/decidim/admin_log/organization_presenter.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+module Decidim
+  module AdminLog
+    # This class holds the logic to present a `Decidim::Organization`
+    # for the `AdminLog` log.
+    #
+    # Usage should be automatic and you shouldn't need to call this class
+    # directly, but here's an example:
+    #
+    #    action_log = Decidim::ActionLog.last
+    #    view_helpers # => this comes from the views
+    #    OrganizationPresenter.new(action_log, view_helpers).present
+    class OrganizationPresenter < Decidim::Log::BasePresenter
```

Comment:
We should consider extending the `OrganizationPresenter` to facilitate the maintainance of module